modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/QuickActions.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import React from 'react';
+
+import {actions} from '../utils/showSidebar';
+
+const HoverItems = ({items, namespace}) => {
+	return (
+		<div className="quick-action-menu">
+			{items.map(({data, href, icon, quickAction}) => (
+				<>
```

Comment:
nit: Fragment probably ins't necessary here. You'd just need to remove the braces around the ternary

Reply:
Nice catch!